rise-google-sheet.html

```diff
@@ -212,7 +212,7 @@
       },
 
       _getDataKey: function() {
-        return DATAKEY_BASE_NAME + "_" + this.key + "_" + this.sheet +
+        return DATAKEY_BASE_NAME + "_" + this.key + "_" + encodeURIComponent( this.sheet ) +
```

Comment:
@rodrigopavezi Fix looks good, do you think this could break any existing presentations that are relying on previous key value without encoding?